web/src/buddy_data.ts

```diff
@@ -234,10 +244,11 @@ export function get_title_data(
     // Since it's not a group, user_ids_string is a single user ID.
     const user_id = Number.parseInt(user_ids_string, 10);
     const person = people.get_by_user_id(user_id);
+    const is_deactivated = !people.is_person_active(user_id);
 
     if (person.is_bot) {
         const bot_owner = people.get_bot_owner_user(person);
-
+        const deactivated_bot_str = is_deactivated ? user_last_seen_time_status(user_id) : "";
```

Comment:
I am not sure about the name (`deactivated_bot_str`) of this variable. Any suggestion regarding the renaming would be highly appreciated :)

Reply:
Yeah, it is mostly that the variable name doesn't really correspond to the function name `user_last_seen_time_status`. I think https://github.com/zulip/zulip/pull/32156/commits/6883c421ad264c86b55952da95782d6d623823fc#r1829159882 might eliminate this conflict.